src/test/java/org/openlmis/stockmanagement/dto/StockCardSummaryV2DtoTest.java

```diff
@@ -100,4 +103,46 @@ public void shouldCompareStockCards() {
         .build();
     assertEquals(1, stockCard1.compareTo(stockCard2));
   }
+
+  @Test
+  public void shouldEnsureComparableContracts() {
```

Comment:
We shouldn't check 4(?) things in a single test. We should split and name the tests appropriately to indicate what each of them tests.

Reply:
Done